docker/tarfile/dest.go

```diff
@@ -168,8 +172,12 @@ func (d *Destination) ReapplyBlob(ctx context.Context, info types.BlobInfo) (typ
 }
 
 func (d *Destination) createRepositoriesFile(rootLayerID string) error {
-	repositories := map[string]map[string]string{
-		d.reference.Name(): {d.reference.Tag(): rootLayerID}}
+	repositories := map[string]map[string]string{}
+	if d.repoTags != nil {
```

Comment:
(Non-blocking: `d.repoTags != nil` is always true due to how `NewDestination` initializes it AFAICT; also, `nil` is a valid slice value, so `range d.repoTags` does the right thing if it were `nil`.  So, this conditional does not need to exist.)

Reply:
right, my bad. Fixed.